sspl_test/run_qa_test.sh

```diff
@@ -48,6 +48,11 @@ flask_help()
 
 pre_requisites()
 {
+    # copy RMQ password to sspl_test/config
+    pw=$($CONSUL_PATH/consul kv get sspl/config/RABBITMQINGRESSPROCESSOR/password)
```

Comment:
@cdeshmukh @thavanathan please confirm if it is okay to assume that all 4 processors will have a common password

Reply:
Yes. But to be safe, let's read from respective processor keys from consul.

Reply:
till last week they were same, now may not, after Provisioner possibly generating separate password for different processor, so better read & use separately

Reply:
Done